### ISSUE TITLE
Request: Change color of stacks count / Cast suggestion

### ISSUE THREAD

**Issue by Tonyleila** [AdiAddons/AdiButtonAuras#165]
Woud be nice if we coud change the color of stacks counts and Cast suggestion.
Thank you!

**Comment by Ripperjack** [comment]
This mod sounds like the one I've been looking for forever. The problem is I don't know lua. I spent a day trying to figure out how to get this very topic to work. On an enhancement shaman, Adibuttonauras cause the free instant cast for spells affected Maelstrom weapon to spangle at the first stack. Stacks have to attain the five max to be beneficial. I need to change it to grab my attention at five stacks, not one. It keeps making think I can use those spells before I can actually use them. This really makes it useless since it makes me have to look up at my buffs to see how many stacks I have, which is what I'm trying to avoid in the first place.

In other words, is there a way to tell it to look for a aura, then when the aura is active to look at how many stacks it has and then at a certain number of stacks, to let us know the spell is able to be used? Or, have one signal (color?) of when the aura is active, then another signal (flash?) when a aura has the needed number of stacks for maximum benefit?

**Comment by Rainrider** [comment]
a5b19dc0c2bbe164645f985c81a0afae3bbe2145 implements the stacks count color change, however this has the limitation that it won't do anything on buttons where the stock ui shows the stack counter (we don't touch default ui elements).

What do you mean by "change color of cast suggestion"?

**Comment by Tonyleila** [comment]
Buffs e.g. Power Word: Fortitude have this cast suggestions where is shows the number of groupmembers that don't have the buff. I'd like to change the color of them :)

**Comment by Rainrider** [comment]
Since a5b19dc0c2bbe164645f985c81a0afae3bbe2145 the count text color is the same as the text color for the countdown timer when above 10 seconds (it is actually above 3 seconds, it is kind of confusing). So the question is, do you want for some reason to have different colors for stacks count and countdown text?

P.S. The number of players missing the buff is represented by the stack counter.

**Comment by Tonyleila** [comment]
Yes stacks count and countdown shoud have different colors, makes it more easy to see what is what. e.g. my countdown text is white -> Yellow -> Red but the stackcount shoud be blue.

**Comment by Rainrider** [comment]
This is easily doable, I will see to it later. However please keep in mind that if the stock ui counter is visible, ABA's own is hidden and we do not touch the color of the default one.

**Comment by Rainrider** [comment]
Ancient
